core/domain/contribution_stats_services.py

```diff
@@ -252,6 +253,8 @@ def get_translation_submitter_total_stats(
             result.
         topic_ids: List[str]|None. List of topic ID(s) to fetch
             contributor stats for.
+        max_days_since_first_activity: int. To get number of users
+            who are active in max_days_since_first_activity.
         max_days_since_last_activity: int. To get number of users
             who are active in max_days_since_last_activity.
```

Comment:
Can you describe what the parameter represents without using the parameter name? Ditto below.

Reply:
done